web/client/themes/default/less/ol.less

```diff
@@ -138,12 +138,20 @@ div.ol-scale-line.ol-unselectable {
     top: 10px;
     position: relative;
 }
+div.ol-scale-line.ol-mobile {
```

Comment:
I would suggest to remove the ol-mobile class and check all the places where the ol-scale-line is used inside the codebase. Most of the changes on the scale line were applied because we wanted it inside the footer so we should put them in the class wrapper of the footer:

```
.mapstore-map-footer {
    // .... all custom classes
}
```
Note that now the position is not correct:

![image](https://github.com/geosolutions-it/MapStore2/assets/19175505/eac387d4-b602-4b62-aa61-2dc4288298c6)


We should position it as we have the leaflet one:

![image](https://github.com/geosolutions-it/MapStore2/assets/19175505/5894c131-1f18-49bb-bc84-91d4500eec6d)

to recap:

- custom style for the mapstore footer needs to be wrapped in the .mapstore-map-footer class
- style for mobile needs to be located on the right position like the one we have for leaflet